CHANGELOG.md

```
@@ -1,3 +1,9 @@
+# [2.6.0](https://github.com/VilnaCRM-Org/user-service/compare/v2.5.0...v2.6.0) (2025-05-28)
+
+### Features
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)